.buildkite/pipeline.gpu.large.yml

```diff
@@ -39,6 +39,8 @@
     - DATA_PROCESSING_TESTING=1 TRAIN_TESTING=1 TUNE_TESTING=1 ./ci/env/install-dependencies.sh
     - pip install -Ur ./python/requirements_ml_docker.txt
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=gpu python/ray/air/...
+    - pip install -U horovod
```

Comment:
Ah instead of manually installing here you can set the `INSTALL_HOROVOD=1` env var for `install-dependencies.sh`

Reply:
oh nice let me change it